code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -541,6 +541,10 @@ datum/recipe/microwave/slimesandwich
 	being_cooked.reagents.del_reagent("toxin")
 	return being_cooked
 
+/datum/recipe/microwave/onionrings
```

Comment:
This could be a fryer special interaction instead of a microwave recipe.

Frankly, since it is literally a single ingredient that you are *probably* frying up (since onion rings are a deep-fried food), I'd argue this should move to the fryer as was done with French fries earlier this year.